models/stock_picking_batch.py

```diff
@@ -237,3 +237,81 @@ def is_valid_location_dest_id(self, location_ref):
 
         return all([pick.is_valid_location_dest_id(location=location)
                     for pick in all_done_pickings])
+
+    @api.multi
```

Comment:
api.mutli and self.ensure_one() ?

Reply:
This is how it is in odoo10 so I haven't changed it.  What should I have it to?

Reply:
I think you can remove @api.multi but check with @ws-mps

Reply:
api.multi can be removed, since I think the decorator by default is api.multi in odoo11. Also it is correct to have api.multi and then have self.ensure_one().